src/test/java/page/GoogleMailCreateNewEmailPage.java

```diff
@@ -44,8 +46,10 @@ public GoogleMailCreateNewEmailPage enterEmailAddressee(String email) {
         return this;
     }
 
-    public GoogleMailCreateNewEmailPage enterEmailSubject(String subject) {
-        emailSubjectTextField.sendKeys(subject);
+    public GoogleMailCreateNewEmailPage enterEmailSubject() {
+//        emailSubjectTextField.sendKeys(subject);
+        WebElement element = emailSubjectTextField;
+        javascriptExecutor.executeScript("arguments[0].value='Message from Webdriver';", element);
```

Comment:
Return the method parameter, please

Reply:
Method parameter (String subject) added, script updated.